iis-common/src/main/java/eu/dnetlib/iis/common/utils/ResourceUtils.java

```diff
@@ -0,0 +1,25 @@
+package eu.dnetlib.iis.common.utils;
+
+import java.net.URISyntaxException;
+import java.util.Objects;
+
+/**
+ * Common resource utilities methods.
+ */
+public class ResourceUtils {
+
+    /**
+     * Returns a resource path with properly handled special characters.
+     *
+     * @param cl   ClassLoader instance for resource extraction.
+     * @param name Path of the resource
+     * @return String representing the path to the resource.
+     */
+    public static String resourcePath(ClassLoader cl, String name) {
```

Comment:
A little bit of nitpicking: I would stick to a methods naming convention and specify the action that is being undertaken (missing an appropriate verb).